tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -367,6 +369,72 @@ async def test_softmax_normalization(
     assert parse_data.get("intent") == intent_ranking[0]
 
 
+@pytest.mark.parametrize(
+    "classifier_params, prediction_min, prediction_max, output_length",
+    [
+        (
+            {RANDOM_SEED: 42, EPOCHS: 1, MODEL_CONFIDENCE: "cosine"},
+            -1,
+            1,
+            LABEL_RANKING_LENGTH,
+        ),
+        (
+            {RANDOM_SEED: 42, EPOCHS: 1, MODEL_CONFIDENCE: "inner"},
+            -1e9,
+            1e9,
+            LABEL_RANKING_LENGTH,
+        ),
+    ],
+)
+async def test_cross_entropy_without_normalization(
+    component_builder: ComponentBuilder,
+    tmp_path: Path,
+    classifier_params: Dict[Text, Any],
+    prediction_min: float,
+    prediction_max: float,
+    output_length: int,
+    monkeypatch: MonkeyPatch,
+):
+    pipeline = as_pipeline(
+        "WhitespaceTokenizer", "CountVectorsFeaturizer", "DIETClassifier"
+    )
+    assert pipeline[2]["name"] == "DIETClassifier"
+    pipeline[2].update(classifier_params)
+
+    _config = RasaNLUModelConfig({"pipeline": pipeline})
```

Comment:
can we limit the epochs please to speed up the tests?

Reply:
The epochs is indeed set to 1 in the parameters of the test. Do you mean to set it somewhere else?

Reply:
nope, all good then